blog/electron-33-0.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Electron 33.0.0
+date: 2024-10-14T00:00:00.000Z
+authors: ckerr
+slug: electron-33-0
+tags: [release]
+---
+
+Electron 33.0.0 has been released! It includes upgrades to Chromium 130.0.6723.44, V8 13.0, and Node 20.18.0.
+
+---
+
+The Electron team is excited to announce the release of Electron 33.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- Added an event to help unlock cryptographic devices when a PIN is needed. [#41205](https://github.com/electron/electron/pull/41205)
+- Extended `navigationHistory` API with 2 new functions for better history management. [#42014](https://github.com/electron/electron/pull/42014)
+- Improved native theme transparency checking. [#42862](https://github.com/electron/electron/pull/42862)
+
+### Stack Changes
+
+- Chromium `130.0.6723.44`
+  - [New in 130](https://developer.chrome.com/blog/new-in-chrome-130/)
+  - [New in 129](https://developer.chrome.com/blog/new-in-chrome-129/)
+- Node `20.18.0`
+  - [Node 20.18.0 blog post](https://nodejs.org/en/blog/release/v20.18.0/)
+  - [Node 20.17.0 blog post](https://nodejs.org/en/blog/release/v20.17.0/)
+- V8 `13.0`
+
+Electron 33 upgrades Chromium from `128.0.6613.36` to `130.0.6723.44`, Node from `20.16.0` to `20.18.0`, and V8 from `12.8` to `13.0`.
+
+### New Features
+
+- Added the event `client-certificate-request-password` to app. It is dispatched when a PIN is needed to unlock a cryptographic device. [#41205](https://github.com/electron/electron/pull/41205)
```

Comment:
Same as above, #42014 didn't actually add a new public event.

Reply:
updated in ed6a3471798bf437821f7b97b08e0cb307739700